python/tests/integration/connectors/memory/test_astradb.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import os
+import time
+
+import numpy as np
+import pytest
+
+import semantic_kernel as sk
+from semantic_kernel.connectors.memory.astradb import AstraDBMemoryStore
+from semantic_kernel.memory.memory_record import MemoryRecord
+
+astradb_installed = True
+
+pytestmark = pytest.mark.skipif(not astradb_installed, reason="astradb is not installed")
```

Comment:
I thought I remembered seeing this handled differently. We can't have `astradb_installed` default to True, and then have `pytest.mark.skipip(...)` with no other logic in between. We need the ability to set `astradb_installed` to false. Please follow a pattern similar to other connector integration tests, as is done [here](https://github.com/microsoft/semantic-kernel/blob/4317f2745e22055eee9eae811d7a4c48d557948d/python/tests/integration/connectors/memory/test_mongodb_atlas.py#L16).

Reply:
We are working on this now.

Reply:
Fixed: astradb_installed is set to false if the memory store or reqs fail to import

Reply:
@moonbox3 @eavanvalkenburg there's a fix for this pushed

Reply:
@hc33brackles, @david-polo, @anomnaco, please run `black .` in the `semantic-kernel/python` directory and push the changes. Once in, I can try and merge to main. Thanks!